docs/slurk_layouts.rst

```diff
@@ -139,7 +139,7 @@ Examples:
 Called when state of currently typing users is changed
 
 Variables:
-  - ``users``: A map of currently typing users, with its id as the key
+  - ``users``: A map of currently typing users, with whose id as the key
```

Comment:
I think jana really did not understand who was meant and didn't intend to replace its by whose.
```suggestion
  - ``users``: A map of currently typing users, with their IDs as keys
```